test/e2e/testsuite/testsuite.go

```diff
@@ -961,9 +961,11 @@ func PodmanCommandIsAvailable() error {
 		csshk = filepath.Join(userHomeDir, ".crc/machines/crc/id_ed25519")
 		dh = "npipe:////./pipe/crc-podman"
 	}
+	/* change linux default to user
 	if runtime.GOOS == "linux" {
 		ch = "ssh://core@192.168.130.11:22/run/user/1000/podman/podman.sock"
 	}
+	*/
```

Comment:
@lilyLuLiu May be you should remove that completely instead just commenting out?